src/utils/dateUtil.ts

```diff
@@ -1,11 +1,13 @@
 export const convertTimestampToDateStr = (timestamp: string): string => {
   const d = new Date(Number(timestamp) * 1000);
-  return formatDate(d.toISOString());
-};
-
-const formatDate = (dateStr: string): string => {
-  const d = dateStr.split("-");
-  return `${d[2].split("T")[0]}-${d[1]}-${d[0]}`;
+  d.getMonth();
```

Comment:
TODO: It will be affected because it is used a lot in other parts of the system, and I need to define other function.

Reply:
fix: ec99ed9